manager/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: "3.7"
+
+services:
+  scylla-manager:
+    image: scylladb/scylla-manager:2.6.0
+    networks:
+      public:
+    depends_on:
+      - scylla-manager-db
+
+  scylla-manager-db:
+    build: .
+    volumes:
+      - scylla_manager_db_data:/var/lib/scylla
+    networks:
+      public:
+    command: --smp 1 --memory 100M
+
+  scylla-node1:
+    container_name: scylla-node1
+    build: .
+    restart: always
+    command: --seeds=scylla-node1,scylla-node2 --smp 1 --memory 750M --overprovisioned 1 --api-address 0.0.0.0
+    volumes:
+      - "./scylla/scylla.yaml:/etc/scylla/scylla.yaml"
```

Comment:
is this really needed?
command should have all setup, so changing/adding scylla.yaml manually is not needed (unless you prove me wrong)
dtto for rackdc properties